os/board/rtl8730e/src/component/wifi/inic/inic_ipc_host_api.c

```diff
@@ -16,20 +16,30 @@
 #include "inic_ipc_cfg.h"
 #include "wifi_ind.h"
 
+#include "rtk_km4log.h"
+#include "osif.h"
+
 /* -------------------------------- Defines --------------------------------- */
 #define CONFIG_INIC_IPC_HOST_API_PRIO 3
 #if defined(CONFIG_ENABLE_WPS) && CONFIG_ENABLE_WPS
 #define HOST_STACK_SIZE 2048
 #else
 #define HOST_STACK_SIZE 1024	// for psp overflow when update group key: jira: https://jira.realtek.com/browse/RSWLANQC-1027
 #endif
+
+#define NP_MAX_QUEUE_SIZE KM4_MAX_QUEUE_SIZE
```

Comment:
why not just use CONFIG_KM4_MAX_QUEUE_SIZE for all?
you created 3 macro pointing to same thing: CONFIG_KM4_MAX_QUEUE_SIZE, KM4_MAX_QUEUE_SIZE, NP_MAX_QUEUE_SIZE. Is it necessary?